config/database.yml

```diff
@@ -20,6 +20,9 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
+  username: postgres
+  password: password
+  host: localhost
```

Comment:
I was just following the pattern of setup for all of the other Firehose apps, but I suppose it would make sense that not listing these credentials explicitly is better practice. I think what we would do here is use an environment variable, but I'm generally not very well-versed in how this interaction between Active Record and Postgres works, especially in development vs production environments. Maybe @lasley has some insight?